components/logo/LogoComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import { useRouter } from "next/router";
+
+const LogoComponent = () => {
+  
+  const { locale } = useRouter();
+  
+  const WEBSITE_URL = 'https://www.indiavaccine.in';  
+   
+  const logoUrl = `/assets/logo/${locale}.svg`;
+  
+  return (
+    <a href={WEBSITE_URL}>
```

Comment:
Rather than pointing to the main domain, we can do this

```suggestion
    <a href={`/${locale}`}>
```

Reply:
make sense,  👍 done